hack/lib/etcd.sh

```diff
@@ -108,6 +108,8 @@ kube::etcd::install() {
     cd "${KUBE_ROOT}/third_party"
     os=$(uname | tr "[:upper:]" "[:lower:]")
     if [[ $(readlink etcd) == etcd-v${ETCD_VERSION}-${os}-* ]]; then
+      kube::log::info "etcd v${ETCD_VERSION} already installed. To use:"
+      kube::log::info "export PATH=$(pwd)/etcd:\${PATH}"
```

Comment:
Instead of suggesting to change your path, perhaps we could just `echo $(readlink etcd)`? We don't actually need to modify the path at all here if this evaluates true.

Reply:
@cblecker fixed as you suggested, PTAL